client/components/forms/docs/example.jsx

```diff
@@ -77,10 +82,14 @@ var FormFields = React.createClass( {
 
 					<FormFieldset>
 						<FormLegend>Form Checkbox</FormLegend>
-						<FormLabel>
-							<FormCheckbox id="comment_like_notification" name="comment_like_notification" />
-							<span>Email me when someone Likes one of my comments.</span>
-						</FormLabel>
+
+						<Checkbox
+							id="comment_like_notification"
```

Comment:
Outside the scope of this pull request, but I wonder whether there's truly a need for an `id` here (or even `name`?).

I've seen some opposition to relying on `props` spread too much, as it obscures what's being passed, but it can be handy for allowing a component to pass along miscellaneous props, especially when it's directly wrapping a DOM node that accepts a variety of attributes (props). In other words, rather than adding [`indeterminate`](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate) (or in this case, `id` or `name`) as a prop of `<Checkbox />` in the rare case it's ever needed, rendering its `<input />` as:

```jsx
<input { ...this.props } type="checkbox" />
```